css/style.css

```diff
@@ -0,0 +1,706 @@
+  padding: 0 20px;
+}
+
+.subscribe-nyt {
+  text-align: center;
+}
+
+.hr-width {
+  width: 75px;
+}
+
+.subscribe-nyt h6 {
+  margin: 0;
+  margin-bottom: -10px;
+  color: #333;
+  font-size: 10px;
+}
+
+.subscribe-nyt a {
+  text-decoration: none;
+  color: #326891;
+  font-size: 10px;
+  font-weight: 600;
+}
+
+.editor-picks {
+  width: 13%;
+
+  /* border: 1px solid red; */
+  position: absolute;
+  right: 12%;
+  top: 1945px;
+}
+
+.editor-picks h5 {
+  margin: 0;
+  font-size: 11px;
+}
+
+.editor-picks hr {
+  margin: 0;
+  color: #e2e2e2;
+}
+
+.picks {
+  display: flex;
+
+  /* border: 1px solid green; */
+  justify-content: space-around;
+  align-items: center;
+}
+
+@media (min-width: 1200px) {
+  .picks {
+    margin: 15px 0;
+  }
+
+  .inflation-theory {
+    height: 80vh;
+  }
+}
+
+.picks img {
+  width: 28%;
+  height: 100%;
+}
+
+.picks p {
+  width: 60%;
+  font-size: 11px;
+  line-height: 12px;
+}
+
+.inflation-theory {
+  width: 60%;
+  height: 95vh;
+  margin: 0 auto;
+  display: grid;
+  grid-template: 100px repeat(4, 1fr) 100px / repeat(4, 1fr);
+  grid-template-areas: "header header header header" "figure1 figure2 figure3 figure4" "caption1 caption2 caption3 caption4" "figure5 figure6 figure7 figure8" "caption5 caption6 caption7 caption8" "footer footer footer footer";
+  grid-gap: 10px;
+}
+
+.header {
+  grid-area: header;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  font-size: 13px;
+}
+
+.caption {
+  font-size: 12px;
+  line-height: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.inflation-theory img {
+  width: 100%;
+}
+
+.inflation-theory .img1 {
+  grid-area: figure1;
+}
+
+.inflation-theory .caption1 {
+  grid-area: caption1;
+}
+
+.inflation-theory .img2 {
+  grid-area: figure2;
+}
+
+.inflation-theory .caption2 {
+  grid-area: caption2;
+}
+
+.inflation-theory .img3 {
+  grid-area: figure3;
+}
+
+.inflation-theory .caption3 {
+  grid-area: caption3;
+}
+
+.inflation-theory .img4 {
+  grid-area: figure4;
+}
+
+.inflation-theory .caption4 {
+  grid-area: caption4;
+}
+
+.inflation-theory .img5 {
+  grid-area: figure5;
+}
+
+.inflation-theory .caption5 {
+  grid-area: caption5;
+}
+
+.inflation-theory .img6 {
+  grid-area: figure6;
+}
+
+.inflation-theory .caption6 {
+  grid-area: caption6;
+}
+
+.inflation-theory .img7 {
+  grid-area: figure7;
+}
+
+.inflation-theory .caption7 {
+  grid-area: caption7;
+}
+
+.inflation-theory .img8 {
+  grid-area: figure8;
+}
+
+.inflation-theory .caption8 {
+  grid-area: caption8;
+}
+
+.footer::before {
+  width: 60px;
+  content: '';
+  display: inline-block;
+  height: 1px;
+  margin: 0;
+  background-color: #000;
+}
+
+.footer {
+  grid-area: footer;
+}
+
+.telescope {
+  margin: 0;
+}
+
+.telescope img {
+  max-width: 100%;
+}
+
+.bottompart {
+  font-size: 0.75rem;
+  line-height: 1rem;
+  color: rgb(102, 102, 102);
+}
+
+.bottompart a {
+  color: rgb(102, 102, 102);
+}
+
+.moreinspace a {
+  color: black;
+  text-decoration: none;
+}
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.color-black{
  color: black;
}
```
**Html file**

```
<div class="color-black">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_